packages/grid/_modules_/grid/models/api/gridSelectionApi.ts

```diff
@@ -9,10 +9,10 @@ export interface GridSelectionApi {
   /**
    * Toggle the row selected state.
    * @param id
-   * @param allowMultiple Default: false = deselect other rows if isSelected is true
    * @param isSelected Default true
+   * @param allowMultiple Default: false = deselect other rows if isSelected is true
    */
-  selectRow: (id: GridRowId, allowMultiple?: boolean, isSelected?: boolean) => void;
+  selectRow: (id: GridRowId, isSelected?: boolean, allowMultiple?: boolean) => void;
```

Comment:
Why do we need this change?

Reply:
Because the TypeScript definition don't match the actual implementation.

https://github.com/mui-org/material-ui-x/blob/348a6731494faebeedebf48266c576d8e353da3e/packages/grid/_modules_/grid/hooks/features/selection/useGridSelection.ts#L102-L107